extensions/http/src/jolie/net/HttpProtocol.java

```diff
@@ -202,6 +203,7 @@
 		private static final String CONTENT_DISPOSITION = "contentDisposition";
 		private static final String DROP_URI_PATH = "dropURIPath";
 		private static final String CACHE_CONTROL = "cacheControl";
+		private static final String STRING_RAW_CONTENT = "stringRawContent";
```

Comment:
I do not like it so much, this new parameter which switches off the correct content parsing. Is it not possible to put these messages under the content type **application/octet-stream**?

If not I would call such a parameter _forceRaw_.